packages/website/tests/rules.spec.ts

```diff
@@ -0,0 +1,32 @@
+import AxeBuilder from '@axe-core/playwright';
+import { expect, test } from '@playwright/test';
+
+test.describe('Rules Page', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/rules');
+  });
+
+  test('Accessibility', async ({ page }) => {
+    await new AxeBuilder({ page }).analyze();
+  });
```

Comment:
😍 thanks for adding this in!

_(I'll continue reviewing soon, just wanted to express appreciation sooner)_